### ISSUE TITLE
It's impossible to move between characters in editor with VoiceOver on MacOs

### ISSUE THREAD

**Issue by sevapopov2** [microsoft/vscode#87851]
Issue Type: <b>Bug</b>

1. Turn on VoiceOver by pressing comand+f5. 
2. Open visual studio code and find the editor. 
3. Try to type the text in editor or open an existing project with the code. 
4. Turn off quick navigation by pressing left and right arrows together. 
5. Move between characters of the code by pressing left and right arrow keys. You will hear the part of the code that is left after pressing left or right arrow and the information what enviroment you are using, in my case it is python 3.8.1. What should be expected is that instead of reading all this information VoiceOver should read the symbol where cursor is. To understand how it should work, open the text edit, type the text and try to move between characters by pressing left and right arrow keys with quick navigation off.

VS Code version: Code 1.41.1 (26076a4de974ead31f97692a0d32f90d735645c0, 2019-12-18T14:57:51.166Z)
OS version: Darwin x64 19.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-5350U CPU @ 1.80GHz (4 x 1800)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off<br>surface_control: disabled_off<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>viz_display_compositor: enabled_on<br>viz_hit_test_surface_layer: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 5, 4|
|Memory (System)|8.00GB (3.05GB free)|
|Process Argv||
|Screen Reader|yes|
|VM|0%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
python|ms-|2019.11.50794

</details>

**Comment by isidorn** [comment]
Yes, this is an electron issue that is fixed in latest. Duplicate of https://github.com/microsoft/vscode/issues/87664
